resources/scripts/components/server/schedules/ScheduleCheatsheetCards.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import tw from 'twin.macro';
+
+export default () => {
+    return (
+        <>
+            <div css={tw`md:w-1/2 h-full bg-neutral-600`}>
+                <div css={tw`flex flex-col`}>
+                    <h2 css={tw`py-4 px-6 font-bold`}>Examples</h2>
+                    <div css={tw`flex py-4 px-6 bg-neutral-500`}>
+                        <div css={tw`w-1/2`}>*/5 * * * *</div>
```

Comment:
Adding every X hours value would be beneficial and is something users struggle when they set minutes to *

If a huge list is an issue, I doubt quarter one is even used by anyone.